src/content/app/genome-browser/components/track-settings-panel/track-settings-views/GeneTrackSettings.tsx

```diff
@@ -20,14 +20,12 @@ import { useAppSelector } from 'src/store';
 import useTrackSettings from '../useTrackSettings';
 
 import SlideToggle from 'src/shared/components/slide-toggle/SlideToggle';
-import GlobalTrackSwitch from './components/global-track-switch/GlobalTrackSwitch';
 
-import {
-  getApplyToAllSettings,
-  getTrackSettingsForTrackId
-} from 'src/content/app/genome-browser/state/track-settings/trackSettingsSelectors';
```

Comment:
If we are getting rid of "this track/all tracks" distinction, we should hunt down the code responsible for that and delete it.

Example:

<img width="1050" alt="image" src="https://user-images.githubusercontent.com/6834224/212856599-89f136ce-b81f-478e-9b79-09fe7c60b786.png">